packages/block-editor/src/components/inserter/block-types-tab.js

```diff
@@ -94,56 +80,39 @@ export function BlockTypesTab( {
 
 	return (
 		<div>
-			{ hasChildItems && (
-				<ChildBlocks rootClientId={ rootClientId }>
```

Comment:
The `ChildBlocks` component seems unused and isn't exported publically, I think it'd be good to go ahead and remove it as well.

Reply:
Good catch!